BACKGROUND.md

```diff
@@ -0,0 +1,239 @@
+# The `"$ref"` Story So Far
+
+[Notable discussions and issues](#notable-discussions-and-issues)
+[Select tools and their features](#select-ools-and-their-features)
+[History of reference and identification keywords](#history-of-reference-and-identification-keywords)
```

Comment:
```suggestion
[Notable discussions and issues](#notable-discussions-and-issues)

[Select tools and their features](#select-tools-and-their-features)

[History of reference and identification keywords](#history-of-reference-and-identification-keywords)
```

This is rendering as a single line, and I think the link on the second item was wrong.

Reply:
Thanks- I made it a proper list and fixed the link.  And also improved the formatting of the discussion/issue links and added the JSON Path one - thanks for that!